### PR TITLE
Include license and warranty statement with solc.

### PR DESCRIPTION
**Pull request by chriseth**
We should probably also clean some headers with this pull request.

### CODE REVIEW

File: solc/license_escaped.h [hunk 1]
```diff
@@ -0,0 +1,676 @@
...
+    This program comes with ABSOLUTELY NO WARRANTY; for details type `show w'.
+    This is free software, and you are welcome to redistribute it
+    under certain conditions; type `show c' for details.
+
+The hypothetical commands `show w' and `show c' should show the appropriate
+parts of the General Public License.  Of course, your program's commands
+might be different; for a GUI interface, you would use an "about box".
+
+  You should also get your employer (if you work as a programmer) or school,
+if any, to sign a "copyright disclaimer" for the program, if necessary.
```
Comment (axic) [inline solc/license_escaped.h]: This is not escaped here (well there are a couple of other instances).

**Comment by axic** [comment]
Wouldn't it be better to have a script during the build process which creates the file out of `LICENSE`?

Escaping with `sed s/"/\\\"/` :)

File: solc/license_escaped.h (hunk 1 above)
Comment (chriseth) [reply solc/license_escaped.h]: It is correctly escaped, this juts seems to be an error in github's syntax hilighting.

**Comment by chriseth** [comment]
@axic indeed, I added such a script with the help of @chfast.

**Comment by chriseth** [comment]
Sorry, took a while but this should now be the working version.

**Review by chfast — APPROVED** [review]
Looks good.

File: solc/CommandLineInterface.cpp [hunk 2]
```diff
@@ -510,8 +519,13 @@ void CommandLineInterface::createFile(string const& _fileName, string const& _da
 bool CommandLineInterface::parseArguments(int _argc, char** _argv)
 {
 	// Declare the supported options.
-	po::options_description desc(
-		R"(solc, the Solidity commandline compiler.
+	po::options_description desc(R"(solc, the Solidity commandline compiler.
+
+Solidity Copyright (C) 2014-2017 Solidity contributors.
```
Comment (axic) [inline solc/CommandLineInterface.cpp]: Do we need the copyright statement here? It is not even in the headers yet.

Comment (chriseth) [reply solc/CommandLineInterface.cpp]: Perhaps we can do without, but the GPL "how to apply the license" suggests this. The thing is that the copyright is just shared by the authors which I think is the default, so perhaps we can just remove it.

Comment (axic) [reply solc/CommandLineInterface.cpp]: I think we should sort this in a separate PR which also adds it to the files. Gives us more time to find the proper wording :)

Comment (axic) [reply solc/CommandLineInterface.cpp]: I guess "Copyright (C) 2014-2017 the Solidity authors and contributors." should be more clear. There seem to be a distinction between the two words, whereas author means the "initial creator", while contributor is anyone else.

**Review by axic — CHANGES_REQUESTED** [review]
Please change the CLI to "Copyright (C) 2014-2017 the Solidity authors and contributors."

**Comment by chriseth** [comment]
Perhaps it is better to just drop the statement. I'm not sure if contributors actually would get the copyright by default (as opposed to just the authors).

**Review by axic — APPROVED** [review]

**Pull request merged**